developer-docs-site/docs/concepts/base-gas.md

```diff
@@ -338,7 +338,7 @@ Hence in practice, payload gas is unlikely to be a concern.
 [BCS sequence specification]:      https://github.com/diem/bcs#fixed-and-variable-length-sequences
 [`gas_meter.rs`]:                  https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas/src/gas_meter.rs
 [`initialize()`]:                  https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/doc/storage_gas.md#0x1_storage_gas_initialize
-[`instr.rs`]:                      https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas/src/instr.rs
+[`instr.rs`]:                      https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/aptos-gas-schedule/src/gas_schedule/instr.rs
```

Comment:
Thanks for catching this! Can we also fix other similar links that got broken by the recent removal of `aptos-gas` crate?

Reply:
sure, #9317 is a start. Would you mind if I change some parametres too since they are different now compared to April version?

Reply:
That would be awesome!